cloud-core/src/main/java/cloud/commandframework/Command.java

```diff
@@ -318,6 +328,32 @@ public Command(
         return new ArrayList<>(this.arguments);
     }
 
+    /**
+     * Return a copy of the command argument array, ignoring flag arguments.
+     *
+     * @return Copy of the command argument array, ignoring flag arguments. This List is mutable.
+     * @since 1.8.0
+     */
+    @API(status = API.Status.EXPERIMENTAL, since = "1.8.0")
+    public @NonNull List<CommandArgument<@NonNull C, @NonNull ?>> nonFlagArguments() {
+        List<CommandArgument<C, ?>> arguments = new ArrayList<>(this.arguments);
+        if (this.flagArgument != null) {
+            arguments.remove(this.flagArgument);
+        }
+        return arguments;
+    }
+
+    /**
+     * Returns the flag argument for this command.
+     *
+     * @return The flag argument for the command, or null if no flags are supported.
```

Comment:
Document that it can return null in the main doc, for return something simple like `flag argument or null` is enough